build/pkgs/rich/dependencies

```diff
@@ -0,0 +1,4 @@
+$(PYTHON) markdown_it_py | $(PYTHON_TOOLCHAIN)
```

Comment:
This seems to require `poetry-core`?

Reply:
Not any more, I've now made it a wheel package, so there's no more build dependency.